great-idea/index.html

```diff
@@ -15,9 +15,103 @@
+           <h2>
+            Get Started
+            </h2>
+        </div>
+        <div>
+          <img class="banner" src="img/header-img.png" alt="Image of a code snippet.">
+        </div>
+        </article>
+      
+          <hr> 
+      
+        <section class="top-section">
+          <div>
+            <h3 class="features">
+              Features
+            </h3>
+            <p class="features-paragraph">
+            Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+            </p>
+          </div>
+          <div>
+            <h3 class="about">
+              About
+            </h3>
+            <p class="about-paragraph">
+              Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+            </p>
+          </div>
+        </section>
+        <div class="center-img">
+          <img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
+        </div>   
+        <section class="other-services">
+          <div class="services">
+            <h3 >
+              Services
+            </h3>
+            <p class="services-paragragh">
+              Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+            </p>
+          </div>
+          <div class="product">
+            <h3>
+              Product
+            </h3>
+            <p class="production-paragraph">
+              Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+            </p>
+          </div>
+          <div class="vision">
+            <h3 >
+              Vision
+            </h3>
+            <p class="vision-paragraph">
+              Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+            </p>
+          </div>
+        </section>
+        
+        <hr>
+      
+        <footer>
+          <h3>Contact</h3>
+          <p class="actual-address">123 Way 456 Street</p><br>
+          <p class="city-state">Somewhere, USA</p>
+          <br>
+          <br>
+          <address tel:"1(888)888-8888"> 1 (888) 888-8888</address>
+          <address mailto:"sales@greatidea.io">sales@greatidea.io</address>
+      
+          <small>Copyright Great Idea! 2018</small>
+        </footer>  
       
-    
+    </div>
   </body>
 </html>
```

Comment:
So I am not seeing any updates to your services html file. I don't know if that was a github error or an oversite on your end, but the services page was part of the assignment.